### ISSUE TITLE
Unable to assign vector if they are calling operator

### ISSUE THREAD

**Issue by hailiang194** [hailiang194/NeoSekaiEngine#6]
Hi I have a script like that
```c
SekaiEngine::Math::Vector2D mouseOffset = m_circle.Center - m_offset;
```
And when I built with MSVC the error show like that
```
error C2440: 'initializing': cannot convert from 'const SekaiEngine::Math::CommonVector<2>' to 'SekaiEngine::Math::Vector2D
```
Is there anyway I can fix that

**Comment by hailiang194** [comment]
Hi, We already saw your problem.
There are 2 ways to fix it
* First: 
```SekaiEngine::Math::CommonVector<2> mouseOffset = m_circle.Center - m_offset;```
. This the the base of ```Vector2D``` but in that case, you can't access to the dimesion value of vector
* Second: 
```SekaiEngine::Math::Vector2D mouseOffset(m_circle.Center.X() - m_offset.X(), m_circle.Center.Y() - m_offset.Y());```. This is not really the convenient way but you can take advantages of ```Vector2D```.

Thanks for reporting us. We're gonna fix it on next update.

**Comment by hailiang194** [comment]
Hello, this bug has been fixed in v0.1.0
